apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -1217,6 +1217,31 @@ type AdvancedProcessorConfig struct {
 	// scheduling controls how the pods are scheduled on nodes.
 	// +optional
 	Scheduling *SchedulingConfig `json:"scheduling,omitempty"`
+
+	// Define secondary networks to be checked for enrichment.
+	// +optional
+	SecondaryNetworks []SecondaryNetwork `json:"secondaryNetworks,omitempty"`
```

Comment:
Can we add a mention here that order matters ?

Reply:
in theory the order should not matter, because the network indexes should make the identification unique. It only matters if the same combination of MAC/IP/interface is reused across several networks, and we don't want that to happen because then there's no guarantee of correctness.

Reply:
added some details to the doc, is that better for you?